src/main/webapp/event-info-page/eventInfo.html

```diff
@@ -17,6 +17,8 @@ <h2>Event Information</h2>
     </br>
     <span>Date:  </span><span id="date">June 22 - August 24</span></br>
     </br>
+    <script>getEventId()</script>
+    <script>loadTasks()</script>
     <body onload="createMap();">
```

Comment:
I don't match the function argument for createMap() in html and javascript but the google maps are loaded as i intended. 
It must be needed more corrections and I will do more tests for the correction.